clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -78,6 +82,21 @@ type Controller struct {
 	ssh util.SSHExecutor
 }
 
+// CompleteConfig completes Prometheus manifest file path config
+func CompleteConfig(p *config.PrometheusConfig) {
```

Comment:
I'd name this function more precisely, e.g. `SetupManifests`. We can always rename it later on if more configuration will be added.

Reply:
I actually like CompleteConfig - it's consistent with many other places across the whole k8s.

Reply:
This seems to be matter of preference. But if `CompleteConfig` common in k8s, then it's fine to leave it as is.

Reply:
Changed as suggested, thanks.